src/logic/currencyValues/api/fetchTokenCurrenciesBalances.ts

```diff
@@ -16,13 +16,9 @@ const fetchTokenCurrenciesBalances = (
   excludeSpamTokens = true,
 ): Promise<AxiosResponse<BalanceEndpoint[]>> => {
   const apiUrl = getTxServiceUrl()
-  const url = `${apiUrl}safes/${safeAddress}/balances/usd/?exclude_spam=${excludeSpamTokens}`
+  const url = `${apiUrl}/safes/${safeAddress}/balances/usd/?exclude_spam=${excludeSpamTokens}`
```

Comment:
@dasanra is there a reason to remove the 3000 limit?

Reply:
@nicosampler `/balances/` and `/balances/usd/` don't have this parameter. This can be checked in tx service docs.

https://safe-transaction.staging.gnosisdev.com/